### PR TITLE
Apply template to Millennium Procedure, add auth types

### PR DESCRIPTION
**Pull request by daliboz**
Apply template to Millennium Procedure, add auth types

### CODE REVIEW

**Comment by averyallen** [comment]
+1

File: content/millennium/dstu2/general-clinical/procedure.md [hunk 1]
```diff
@@ -36,11 +56,25 @@ Notes:
...
+    GET https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Procedure?patient=1316020
+
+#### Response
 
 <%= headers status: 200 %>
 <%= json(:dstu2_procedure_bundle) %>
 
+### Errors
+
+The common [errors] may be returned.
```
Comment (JFisk42) [inline content/millennium/dstu2/general-clinical/procedure.md]: Does it make sense to link all the common errors when some aren't accessible for a given resource? (409/422 are only for update/create etc.)

Comment (daliboz) [inline content/millennium/dstu2/general-clinical/procedure.md]: Yeah, we generally don't call out in a specific resource unless it's an exception (an error it returns that isn't "normal").

Comment (JFisk42) [inline content/millennium/dstu2/general-clinical/procedure.md]: +1

File: content/millennium/dstu2/general-clinical/procedure.md [hunk 2]
```diff
@@ -36,11 +56,25 @@ Notes:
 
   - The `date` parameter must have a time, may be provided up to two times, and must use the `ge` or `le` prefixes. When provided twice, the lower value must have the `ge` prefix and the higher value must have the `le` prefix.
 
-### Response
+### Headers
+
+ <%= headers %>
+
+### Example
```
Comment (JFisk42) [inline content/millennium/dstu2/general-clinical/procedure.md]: This is outside the scope of this code review but I just wanted to note that the header sizes for Example and Request/Response look exactly the same in my browser. Request/Response don't generate a TOC link, but it looks slightly odd to me.

**Comment by JFisk42** [comment]
+1

File: content/millennium/dstu2/general-clinical/procedure.md (hunk 2 above)
Comment (MaxPhilips) [inline content/millennium/dstu2/general-clinical/procedure.md]: We have a task logged for this

**Pull request merged**